app/views/core_induction_programmes/mentor_materials/show.html.erb

```diff
@@ -1,9 +1,6 @@
 <div class="govuk-grid-row">
-  <div class="govuk-grid-column-two-thirds">
```

Comment:
If you don't want `govuk-grid-column-two-thirds`, use `govuk-grid-column-full` :)